File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.py
def solution(n):
    fib = [0] * (n + 1)
    fib[1] = 1
    index = 2
    
    while not fib[n]:
        fib[index] = fib[index - 2] + fib[index - 1]
        index += 1
        
    return fib[n] % 1234567

File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.js
function solution(n) {
    let answer = 0;
    const fibo = [0, 1];
    
    while (fibo.length !== n + 1) {
        fibo.push(fibo[fibo.length - 1]  % 1234567 + fibo[fibo.length - 2]  % 1234567)
    }
    return fibo[fibo.length - 1] % 1234567;
}